core/eolearn/tests/test_core_tasks.py

```diff
@@ -55,8 +55,13 @@
 def patch_fixture():
     patch = EOPatch()
     patch.data["bands"] = np.arange(2 * 3 * 3 * 2).reshape(2, 3, 3, 2)
+    patch.data["CLP"] = np.arange(2 * 3 * 3 * 1).reshape(2, 3, 3, 1)
+    patch.mask["CLM"] = np.arange(2 * 3 * 3 * 1).reshape(2, 3, 3, 1)
```

Comment:
now all of them are doubled :thinking:  and they all contain the same kind of data, so if CLM and CLP are mixed up, the test wont discover it. Use `np.full` and things like that, vary shapes a bit.

Reply:
Updated

Reply:
much better :D
I would however have for instance 3 `mask_timeless` features
and maybe a single `mask` feature? :thinking:  because now there are always multiple features for each feature type

Reply:
Updated need to add instance in data so marge on other axis then `-1` can be tested.